src/clr-angular/forms/datepicker/providers/locale-helper.service.ts

```diff
@@ -15,6 +15,8 @@ import {
 } from '@angular/common';
 import { Inject, Injectable, LOCALE_ID } from '@angular/core';
 
+export type DayOfWeekType = { day: string; narrow: string };
```

Comment:
Just to be more consistent with the codebase and objects could we make this an `interface` instead of a type unless there is a particular reason to use `type`?

Reply:
I first intended to only use it internally in the component, but that was too short-sighted of an expectation. Moving to an interface, thanks.

Reply:
done